src/Modix/Program.cs

```diff
@@ -141,10 +141,10 @@ private static void ConfigureServices(WebApplicationBuilder builder, IConfigurat
                 .AddModix(configuration);
 
             builder.Services.AddMvc(d => d.EnableEndpointRouting = false)
-                .AddNewtonsoftJson(options =>
+                .AddJsonOptions(options =>
                 {
-                    options.SerializerSettings.Converters.Add(new StringEnumConverter());
-                    options.SerializerSettings.Converters.Add(new StringULongConverter());
+                    options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
+                    options.JsonSerializerOptions.Converters.Add(new StringULongConverter());
```

Comment:
FWIW, I don't think this is needed
In the WASM rewrite I completely removed newtonsoft on the ASP.NET side of things including the converters

I assume this is a remnant from the Vue days.

https://github.com/discord-csharp/MODiX/pull/1024/files#diff-141af68f60363a161017e6a0436aee53c10f47342727cec85b45296ce905db14L142-L148

Reply:
let me update the PR description to clarify the context